orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ResizeServerGroupTask.groovy

```diff
@@ -40,10 +39,23 @@ class ResizeServerGroupTask extends AbstractServerGroupTask {
   @Autowired
   TrafficGuard trafficGuard
 
-  Map getAdditionalStageOutputs(Stage stage, Map operation) {
+  Map getAdditionalContext(Stage stage, Map operation) {
     [capacity: operation.capacity]
   }
 
+  /**
+   * Track the _original_ capacity of the server group being resized in case it needs to be subsequently restored.
+   */
+  @Override
+  Map<String, Object> getAdditionalOutputs(Stage stage, Map operation) {
+    def originalCapacityKey = "originalCapacity.${operation.serverGroupName}".toString()
```

Comment:
Supports a similar pattern to that used in `AbstractAwsScalingProcessTask`

```
def processes = filterProcesses(
      targetServerGroup,
      (stage.context.processes ?: stage.context["scalingProcesses.${asgName}" as String]) as List<String>
    )
```

A `ResizeServerGroupStage` can optionally look for original capacity and use it to unpin the `min`.